tests/suites/authorized_keys/bootstrap.sh

```diff
@@ -28,10 +28,11 @@ run_bootstrap_authorized_keys_loaded() {
 	ssh-keygen -t ed25519 -f "$extra_key_file" -C "isgreat@juju.is" -P ""
 
 	HOME="${SUB_TEST_DIR}"
+	JUJU_DATA="${juju_home_dir}"
 	bootstrap_additional_args=(--config "'authorized-keys=$(cat ${extra_key_file_pub})'")
 	export BOOTSTRAP_ADDITIONAL_ARGS="${bootstrap_additional_args[*]}"
 	export BOOTSTRAP_REUSE=false
-	bootstrap "authorized-keys-default" "$log_file"
+	bootstrap "authorized-keys-loaded" "$log_file"
```

Comment:
Why change the controller name?

Reply:
It was wrong and causing another issue. It should reflect the name of the test. Just a bad copy and paste i fixed as a drive by.